service/device/registry/api/src/main/java/org/eclipse/kapua/service/device/registry/connection/DeviceConnectionService.java

```diff
@@ -71,4 +71,7 @@ void connect(DeviceConnectionCreator creator)
      */
     void disconnect(KapuaId scopeId, String clientId)
             throws KapuaException;
+
+    DeviceConnection findByUserId(KapuaId scopeId, KapuaId userId ) 
```

Comment:
Please don't add this method. Add a `private` method in `GwtUserServiceImpl` that performs this query instead.